<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shop</class>
 <widget class="QDialog" name="shop">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabhistory">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="wowtab">
    <attribute name="title">
     <string>World of Warcraft</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gametime</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_30">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>30d 15$</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_90">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>90d 30$</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_120">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>120d 60$</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buy_button">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expansions 20$</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tbc_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>99</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/tbc.png</normaloff>:/tbc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="wotlk_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/wotlk.png</normaloff>:/wotlk.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cata_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/cata.png</normaloff>:/cata.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="mop_button">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/mop.png</normaloff>:/mop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="wod_button">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>240</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/wod.png</normaloff>:/wod.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;World of Warcraft&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wow_base">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy Base Game</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>386</x>
       <y>86</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>40$</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="d3tab">
    <attribute name="title">
     <string>Diablo II</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Diablo III&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buy_ros">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>151</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/ros.png</normaloff>:/ros.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expansions $40</string>
     </property>
    </widget>
    <widget class="QPushButton" name="d3_base">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy Base Game</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>$40</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sc2tab">
    <attribute name="title">
     <string>Starcraft II</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Starcraft II&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buy_wol">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>141</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/wol.png</normaloff>:/wol.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="buy_hts">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>141</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/hts.png</normaloff>:/hts.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="sc3_base">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy Base Game</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expansions $40</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>$20</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hstab">
    <attribute name="title">
     <string>Hearthstone</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Hearthstone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buy_nax">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>361</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/nax.jpg</normaloff>:/nax.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>360</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="hs_base">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Acquire Base Game</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Free!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expansion $10</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="balancetab">
    <attribute name="title">
     <string>Balance</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>240</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Balance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/bnet.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Battle.net Balance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_balance">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="card_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Credit Card</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="code_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Please  enter your Credit Card code below&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Transactions</string>
    </attribute>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Latest Transactions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Balance Spent</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="total_edit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>126</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="transactions_edit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>291</width>
       <height>251</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabhistory</tabstop>
  <tabstop>rb_30</tabstop>
  <tabstop>rb_120</tabstop>
  <tabstop>buy_button</tabstop>
  <tabstop>tbc_button</tabstop>
  <tabstop>rb_90</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
